### ISSUE TITLE
[Bug] Error firebase Java class com/google/firebase/crashlytics/ndk/FirebaseCrashlyticsNdk not found

### ISSUE THREAD

**Issue by Guidanel** [firebase/quickstart-unity#1194]
### [REQUIRED] Please fill in the following fields:

  * Unity editor version: 2020.3.23f1
  * Firebase Unity SDK version: 8.6.2
  * Source you installed the SDK: .unitypackage
  * Problematic Firebase Component: Crashlytics
  * Other Firebase Components in use: Analytics, Auth, Firestore
  * Additional SDKs you are using: Google Play Games
  * Platform you are using the Unity editor on: Windows
  * Platform you are targeting: Android
  * Scripting Runtime: IL2CPP

### [REQUIRED] Please describe the issue here:

I know that #1177 fixed this issue for Firebase SDK 8.6.1, however the issue happens to me with the new version. CrashlyticsDependencies.xml correctly points to 18.2.4 and there is com.google.firebase.firebase-crashlytics-ndk-18.2.4.aar in my Plugins/Android folder.

Logs:

```
2021/11/27 09:52:08.385 16027 16071 Error firebase Java class com/google/firebase/crashlytics/ndk/FirebaseCrashlyticsNdk not found.  Please verify the AAR which contains the com/google/firebase/crashlytics/ndk/FirebaseCrashlyticsNdk class is included in your app.
2021/11/27 09:52:08.413 16027 16071 Error Unity Java class com/google/firebase/crashlytics/ndk/FirebaseCrashlyticsNdk not found.  Please verify the AAR which contains the com/google/firebase/crashlytics/ndk/FirebaseCrashlyticsNdk class is included in your app.
2021/11/27 09:52:08.413 16027 16071 Error Unity Firebase.Crashlytics.CrashlyticsInternalPINVOKE:FirebaseCrashlyticsInternal_GetInstance__SWIG_0(HandleRef, Int32&)
2021/11/27 09:52:08.413 16027 16071 Error Unity Firebase.Crashlytics.FirebaseCrashlyticsInternal:GetInstance(FirebaseApp, InitResult&)
2021/11/27 09:52:08.413 16027 16071 Error Unity Firebase.Crashlytics.AndroidImpl:.ctor()
2021/11/27 09:52:08.413 16027 16071 Error Unity Firebase.Crashlytics.Impl:Make()
2021/11/27 09:52:08.413 16027 16071 Error Unity Firebase.Crashlytics.PlatformAccessor:.cctor()
2021/11/27 09:52:08.413 16027 16071 Error Unity Firebase.Crashlytics.Crashlytics:Initialize()
2021/11/27 09:52:08.413 16027 16071 Error Unity System.Reflection.MonoMethod:Invoke(Object, BindingFlags, Binder, Object[], CultureInfo)
2021/11/27 09:52:08.413 16027 16071 Error Unity Firebase.FirebaseApp:InitializeCrashlyticsIfPresent()
2021/11/27 09:52:08.413 16027 16071 Error Unity System.Func`1:Invoke()
2021/11/27 09:52:08.413 16027 16071 Error Unity Firebase.FirebaseApp:CreateAndTrack(CreateDelegate, FirebaseApp)
2021/11/27 09:52:08.413 16027 16071 Error Unity FirebaseManager:<Start>b__12_0(Task`1)
2021/11/27 09:52:08.413 16027 16071 Error Unity System.Action`1:Invoke(T)
2021/11/27 09:52:08.413 16027 16071 Error Unity Firebase.Extensions.<ContinueWithOnMainThread>c__AnonStorey9:<>m__0()
2021/11/27 09:52:08.413 16027 16071 Error Unity System.Func`1:Invoke()
2021/11/27 09:52:08.413 16027 16071 Error Unity Firebase.<RunAsync>c__AnonStorey1`1:<>m__0()
2021/11/27 09:52:08.413 16027 16071 Error Unity System.
2021/11/27 09:52:08.413 16027 16071 Error firebase clazz
2021/11/27 09:52:08.416 16027 16071 Error Unity clazz
2021/11/27 09:52:08.416 16027 16071 Error Unity Firebase.Crashlytics.CrashlyticsInternalPINVOKE:FirebaseCrashlyticsInternal_GetInstance__SWIG_0(HandleRef, Int32&)
2021/11/27 09:52:08.416 16027 16071 Error Unity Firebase.Crashlytics.FirebaseCrashlyticsInternal:GetInstance(FirebaseApp, InitResult&)
2021/11/27 09:52:08.416 16027 16071 Error Unity Firebase.Crashlytics.AndroidImpl:.ctor()
2021/11/27 09:52:08.416 16027 16071 Error Unity Firebase.Crashlytics.Impl:Make()
2021/11/27 09:52:08.416 16027 16071 Error Unity Firebase.Crashlytics.PlatformAccessor:.cctor()
2021/11/27 09:52:08.416 16027 16071 Error Unity Firebase.Crashlytics.Crashlytics:Initialize()
2021/11/27 09:52:08.416 16027 16071 Error Unity System.Reflection.MonoMethod:Invoke(Object, BindingFlags, Binder, Object[], CultureInfo)
2021/11/27 09:52:08.416 16027 16071 Error Unity Firebase.FirebaseApp:InitializeCrashlyticsIfPresent()
2021/11/27 09:52:08.416 16027 16071 Error Unity System.Func`1:Invoke()
2021/11/27 09:52:08.416 16027 16071 Error Unity Firebase.FirebaseApp:CreateAndTrack(CreateDelegate, FirebaseApp)
2021/11/27 09:52:08.416 16027 16071 Error Unity FirebaseManager:<Start>b__12_0(Task`1)
2021/11/27 09:52:08.416 16027 16071 Error Unity System.Action`1:Invoke(T)
2021/11/27 09:52:08.416 16027 16071 Error Unity Firebase.Extensions.<ContinueWithOnMainThread>c__AnonStorey9:<>m__0()
2021/11/27 09:52:08.416 16027 16071 Error Unity System.Func`1:Invoke()
2021/11/27 09:52:08.416 16027 16071 Error Unity Firebase.<RunAsync>c__AnonStorey1`1:<>m__0()
2021/11/27 09:52:08.416 16027 16071 Error Unity System.Action:Invoke()
2021/11/27 09:52:08.416 16027 16071 Error Unity Firebase.ExceptionAggregator:Wrap(Action)
2021/11/27 09:52:08.416 16027 16071 Error Unity Firebase.Dispatcher:PollJobs()
2021/11/27 09:52:08.416 16027 16071 Error Unity Firebase.Platform.FirebaseHandler:Update()
2021/11/27 09:52:08.416 16027 16071 Error Unity 
2021/11/27 09:52:08.416 16027 16071 Error firebase Java class com/google/firebase/crashlytics/ndk/FirebaseCrashlyticsNdk not found.  Please verify the AAR which contains the com/google/firebase/crashlytics/ndk/FirebaseCrashlyticsNdk class is included in your app.

```

**Comment by Guidanel** [comment]
Fixed in the comments of #1179 (my bad I didn't see that earlier)
https://github.com/firebase/quickstart-unity/issues/1179#issuecomment-976412970

**Comment by paulinon** [comment]
Hi @Guidanel,

I see that you've found a solution to your issue. With that, I'll be closing this for now. In the event that a new issue arises, let us know by leaving a comment so that we can identify the cause of this.
